src/app/components/character-count/index.html

```diff
@@ -0,0 +1,284 @@
+<sky-tabbed-demo-page
+  pageTitle="Character Count"
+>
+
+  <sky-demo-page-summary>
+  The character count component is an extension for text input fields with character limits.
+  </sky-demo-page-summary>
+
+  <div
+    class="sky-margin-stacked-content-section"
+  >
+    <sky-demo-page-example>
+      <sky-character-count-demo>
+      </sky-character-count-demo>
+    </sky-demo-page-example>
+  </div>
+
+  <sky-tabset
+    id="content-tabs"
+  >
+
+    <sky-tab
+      tabHeading="Design"
+    >
+
+      <sky-demo-page-content
+        sectionHeading="Usage"
+      >
+
+        <h3
+          class="sky-text-success"
+        >
+          Use when
+        </h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              Use character count when a user has a high likelihood of being impacted by a character limit. This could be because technical requirements make the limit low, or because user input is freeform and could be very long.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-1.png"
+              />
+              <div
+                class="caption caption-success sky-text-success"
+              >
+                Do use character count on input fields with character limits that a user could realistically exceed.
+              </div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+
+        <h3
+          class="sky-text-danger"
+        >
+          Don't use when
+        </h3>
+
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              Don’t use character count on inputs where the character limit is apparent or implied by the data type.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-2.png"
```

Comment:
I think the image here could be slightly confusing. Maybe it doesn't matter, but the indicator shows that the character limit is 10 while the default value in the field has 12 characters counting the parentheses. Should we update the limit to reflect the parentheses? When I test the phone field, it doesn't add the parenthesis by default, so I would think they would count in the character count.

Reply:
Updated image, and I made it 13 to include the dash.